### ISSUE TITLE
Axis not scaling correctly?

### ISSUE THREAD

**Issue by matt-storr** [leo-holanda/logcharts.io#6]
Added two data sets; CPU Power (watts), CPU Package Temp.  The x-axis correctly displays the first data set values, but adding the second and selecting it doesn't re-scale the axis, causing the values to be misinterpreted

**Comment by matt-storr** [comment]
![image](https://github.com/leo-holanda/logcharts.io/assets/33256692/699956e1-3a8c-47fd-a392-83ecadfd6a4a)

**Comment by matt-storr** [comment]
Am I misunderstanding how the x-axis should be interepreted, or is there some normalisation happening?  I would have thought at least that selecting the relevant data set, the axis should re-scale itself?

**Comment by leo-holanda** [comment]
Hi @matt-storr! Sorry for the delay.

The re-scale thing happens only when the first selector is changed. So, if you change the first selector, which holds the CPU Power data set, to another data set, it will re-scale and show the new values on the x and y-axis. I believe the x-axis values are always the same for any data set since the log collects all values at the same second. Because of that, only the brush can influence the x-axis values since it can get bigger or smaller.

Selecting the radio button only marks which selector will change when you click the buttons on the left. And only when the first  selector is selected that it will re-scale. I should add an indicator that says it as this misconception can happen again.

Selecting the second or third or any selector that it`s not the first and clicking on the data set button will redraw its line to show the new data but it will not re-scale the x-axis. 

So, in summary, the x-axis is the same for all data sets and only change when the first selector data set change. The y-axis is tied to each data set and it always changes when the data-set changes. But the y-axis values that you see are always tied to the first selector data set. The other ones are hidden, lets say.

Please let me know if I understand your question correctly and if the answer is what you were expecting.
